source/slang/type-layout.h

```diff
@@ -328,6 +328,24 @@ class TypeLayout : public Layout
     // the space storing it in the above array
     UInt uniformAlignment = 1;
 
+        /// An item that is conceptually owned by this type, but is pending layout.
+        ///
+        /// When a type contains interface/existential fields (recursively), the
+        /// actual data referenced by these fields needs to get allocated somewhere,
+        /// but it cannot go inline at the point where the interface/existential
+        /// type appears, or else 
```

Comment:
or else... sounds like a threat

Reply:
Hah. Clearly got distracted by something else while writing the comment.